widgets/serializers/rss.py

```diff
@@ -63,3 +85,18 @@ def get_json(self, instance):
                 for entry in sorted_feed[:display_limit]
             ],
         }
+
+    def save(self, **kwargs):
+        """Saves the widget settings"""
+        instance = super().save(**kwargs)
+
+        try:
+            url = instance.configuration["url"]
+
+            # force a cache refresh immediately after a successful save by invalidating and then loading it
+            _fetch_rss.invalidate(url)
+            _fetch_rss(url)
```

Comment:
It can, and I originally had this, but `pylint` chokes on it so the line ended up needing to be:

```python
_fetch_rss(url, _refresh=True)  ## pylint: disable=unexpected-keyword-arg
```

So I ended up going with the more verbose version here over disabling the pylint check.